src/main/java/com/uber/cadence/internal/common/WorkflowExecutionUtils.java

```diff
@@ -553,60 +553,6 @@ public static WorkflowExecutionInfo describeWorkflowInstance(
     return instanceMetadata;
   }
 
-  /** Returns workflow instance history in a human readable format. */
```

Comment:
Why is it removed?

Reply:
Because it's unused and in an internal library. There's another overload which prints the history in a pretty format given the history. Do we need a print method that fetches the history on it's own? I can add it back but i'm not sure i see the purpose.

Reply:
Let's keep it as it can be used by tools. For example https://github.com/uber/cadence-java-samples/blob/master/src/main/java/com/uber/cadence/samples/common/WorkflowExecutionHistoryPrinter.java

Reply:
ok, reverted